objcgen/objcgenerator.cs

```diff
@@ -151,15 +151,17 @@ protected override void Generate (Type t)
 			var native_name = GetTypeName (t);
 			headers.WriteLine ();
 			headers.WriteLine ($"// {t.AssemblyQualifiedName}");
-			headers.WriteLine ($"@interface {native_name} : {GetTypeName (t.BaseType)}");
```

Comment:
Yes, more changes are needed for that, but I wanted to get this in since other people were running into issues I'd already fixed here (-o didn't work).